src/main_cfe_aorc_pet.c

```diff
@@ -6,8 +6,8 @@
 #include "../include/bmi_cfe.h"
 #include "../forcing_code/include/pet.h"
 #include "../forcing_code/include/bmi_pet.h"
-#include "../forcing_code/include/aorc.h"
-#include "../forcing_code/include/bmi_aorc.h"
+#include "../extern/forcing_code/include/aorc.h"
+#include "../extern/forcing_code/include/bmi_aorc.h"
```

Comment:
you also need to update your bmi_aorc.c. The existing aorc_bmi.c file in your repo is outdated as it expects rain in units of mm/sec while our forcing data is in mm/hr which leads to unexpected results. So please update your aorc_bmi.c file by copying cfe/forcing_code/src/bmi_aorc.c to your aorc_bmi repo.